mppt/GlobalMPPTAlgos/GlobalMPPTAlgorithm.cpp

```diff
@@ -0,0 +1,70 @@
+/**
+ * Maximum Power Point Tracker Project
+ * 
+ * File: GlobalMPPTAlgorithm.cpp
+ * Author: Afnan Mir
+ * Organization: UT Solar Vehicles Team
+ * Created on: June 12th, 2021
+ * Last Modified: 06/12/2021
+ * 
+ * File Discription: This cpp file implements the GlobalMPPTAlgorithm class, which
+ * is the parent class of the Global MPPT algroithms.
+ */
+
+#pragma once
```

Comment:
`#pragma` and `#ifdef` guards should only present in header files.
http://faculty.cs.niu.edu/~mcmahon/CS241/c241man/node90.html
https://www.avrfreaks.net/forum/pragma-once-vs-ifndef